docs/Installation_Guide.md

```diff
@@ -212,13 +212,9 @@ Make sure to exit the postgres client before proceeding to the next steps:
 
 Change directory to where you can find the **manage.py** file (this is located in the top level directory for the project). If you are installing the project on the VM, the project will be located within the **/vagrant/vms** directory.
 
-To view the sql commands that will be generated from syncdb, run the command:
-
-    python manage.py sqlall app_name_here
-
-To generate the database tables that correspond to the Django models, run the command:
-
-    python manage.py syncdb
+To generate the database tables 
+Run the following command:
+    python manage.py migrate
```

Comment:
Md needs an extra empty line to display a newline, also put this command in codeticks (`).

Reply:
add a newline in between and put the command in backtick (`)
@Tlazypanda

Reply:
Done @Monal5031

Reply:
codeticks?